.gitignore

```diff
@@ -53,3 +53,5 @@ GRTAGS
 GSYMS
 GTAGS
 .mypy_cache/
+# TODO(#7152): remove this after the python 3 migration!
+/.pants-python-version
```

Comment:
Driveby comment, sorry. But: could maybe put this under `.pants.d`?

Reply:
Oooh, good call actually. @Eric-Arellano since we're not planning to use the `.pants-python-version` file in the setup repo would `.pants.d/.pants-python-version` make sense here?

Reply:
Yes, very good idea! Always an advocate for less at the root level.

Reply:
Done!